src/primitives/transaction.h

```diff
@@ -460,6 +464,11 @@ struct CMutableTransaction
         SerializeTransaction(*this, s, ser_action);
     }
 
+    template <typename Stream>
+    CMutableTransaction(deserialize_type, Stream& s) {
```

Comment:
Right now there's some code redundancy - the only deserialize_type is deserialize,
so it's not used like here.
But I guess people think there may be in the future.